pyexchange/coinbene.py

```diff
@@ -0,0 +1,315 @@
+# This file is part of Maker Keeper Framework.
+#
+# Copyright (C) 2019 grandizzy
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import hashlib
+import logging
+import requests
+import json
+import time
+
+from typing import Optional, List
+from pprint import pformat
+
+from pymaker import Wad
+from pymaker.util import http_response_summary
+from pyexchange.api import PyexAPI
+
+
+class Order:
+    def __init__(self,
+                 order_id: str,
+                 pair: str,
+                 is_sell: bool,
+                 price: Wad,
+                 amount: Wad,
+                 filled_amount: Wad):
+
+        assert(isinstance(pair, str))
+        assert(isinstance(is_sell, bool))
+        assert(isinstance(price, Wad))
+        assert(isinstance(amount, Wad))
+        assert(isinstance(filled_amount, Wad))
```

Comment:
Happy to see this naming convention.  When doing okex, I renamed _deal_amount_ to _filled_amount_.